createSpotifyPlaylist.py

```diff
@@ -0,0 +1,217 @@
+"""
+This module is used to create a spotify playlist
+"""
+import os
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+
+
+class CreateSpotifyPlaylist:
+    """
+    TODO: add class-level docstring
+    """
+    def __init__(self):
+        """Init function
+
+        This function will call the class function `auth_spotify` and create a list for later use.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+
+        """
+        self.spotify_client = self.auth_spotify()
+        self.popular_tracks_list = []
+
+
+    @classmethod
+    def auth_spotify(cls):
+        """Functio to go authenticate spotify
```

Comment:
Use consistent quotes:
~~~python
"""
Some block text
"""
~~~

Reply:
also, spelling

Reply:
I saw on the numpy documentation that the docstring comments looked like
~~~python

"""Comment

More comments......

""""
~~~

Should I just remove the first line comment?